Environment-Definitions/BicepTemplates/REAME.md

```diff
@@ -0,0 +1,14 @@
+# Bicep Deployments
```

Comment:
typo in filename :) 
should be README.md

Reply:
fixed!